### ISSUE TITLE
Low-Memory Neural Network Training: A Technical Report

### ISSUE THREAD

**Issue by icoxfog417** [arXivTimes/arXivTimes#1202]
## 一言でいうと

DNNを省メモリで学習する手法と、その効果をまとめた研究。Sparsity(DNNは実際貢献する重みが少ないため、重みをもつものをまとめ効率化)、Low precision(低精度演算)、Microbatching(ミニバッチを分割し勾配計算)、Gradient checkpointing(forward時特定箇所の勾配だけキープ)の4つを検証している

### 論文リンク

https://arxiv.org/abs/1904.10631

### 著者/所属機関

Nimit Sharad Sohoni, Christopher Richard Aberger, Megan Leszczynski, Jian Zhang, Christopher Ré

### 投稿日付(yyyy/MM/dd)

2019/4/24
